Update ccxt package version in requirements.txt

openstake-net opened the pull request:
ccxt==1.87.48 cause deployment issue with the below error in google cloud function: ERROR: No matching distribution found for ccxt==1.87.48; Error ID: c84b3231 

I updated requirement.txt with cctx:2.6.22 and deployed it successfully.

openstake-net committed:
Update requirements.txt

ccxt==1.87.48 cause deployment issue with the below error in google cloud function:
ERROR: No matching distribution found for ccxt==1.87.48; Error ID: c84b3231 

I updated requirement.txt with cctx:2.6.22 and deployed it successfully.

RogerKSI commented:
Hi, from here (https://pypi.org/project/ccxt/#history), it seems ccxt will always keep versions that end with .1 (X.X.1).
Could you try it with ccxt version 2.6.1 to prevent it from happening again in the future?

openstake-net committed:
Update requirements.txt

tested with ccxt=2.6.1 and google function deployment is completed successfully as well. Updated the pull request accordingly.

openstake-net commented:
tested with ccxt=2.6.1 and google function deployment is completed successfully as well. Updated the pull request accordingly.

openstake-net commented:
We started BAND Chain validation a few months back and would like to have some delegation from BAND Foundation. Thanks, @RogerKSI
